File: platforms/kotlin/examples/core/texture_mip_chain/level_count.kt
import org.fragmentcolor.*

val pixels = Array(8 * 8 * 4) { 0 }
val chain = TextureMipChain.prepare((
    pixels.asSlice(),
    TextureFormat.Rgba8UnormSrgb,
    arrayOf(8, 8),
))
val count = chain.levelCount()
val _ = count

File: platforms/python/examples/core/texture_mip_chain/level_count.py
from fragmentcolor import TextureFormat, TextureMipChain

pixels = [0] * (8 * 8 * 4)
chain = TextureMipChain.prepare(pixels, TextureFormat.Rgba8UnormSrgb, [8, 8])
count = chain.level_count()
_ = count
